starter_code/script.js

```diff
@@ -26,14 +53,17 @@ function evaluate(evt) {
 
     if ( 'user' === winner ) {
         $('#message').delay(50).text('You have won!, Click a box to play again');
+        userPoint ++;
     } else if ( winner === 'draw' ) {
         $('#message').delay(50).text('Draw! Click a box to play again');
     } else {
         $('#message').delay(50).text('You have lost!, Click a box to play again');
+        aiPoint++; 
     }
+    setScore();
 }
 
 // This function runs on page load
 $(document).ready(function(){
-
+    
```

Comment:
seems like you're only missing binding the `evaluate` function to the `click` event